src/objects/Trigger_Handler__c.object

```diff
@@ -89,6 +89,16 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Exclude_For_These_Usernames__c</fullName>
```

Comment:
@smarovich we need string review for this field name and description along with the example docs that Mike forwarded you. This won't go in until next sprint so no huge rush. Thanks!